server/services/mapping_service.py

```diff
@@ -198,8 +198,8 @@ def generate_gpx(project_id: int, task_ids_str: str, timestamp=None):
                     ET.SubElement(trkseg, 'trkpt', attrib=dict(lon=str(point[0]), lat=str(point[1])))
 
                     # Append wpt elements to end of doc
-                    # wpt = ET.Element('wpt', attrib=dict(lon=str(point[0]), lat=str(point[1])))
-                    # root.append(wpt)
+                    wpt = ET.Element('wpt', attrib=dict(lon=str(point[0]), lat=str(point[1])))
```

Comment:
I suspect these were commented out in order to avoid performance issues with loading the Mapillary boundaries into JOSM. We will see if that is the case.